tests/python/test_sync_base.py

```diff
@@ -73,31 +70,7 @@ def setUp(self):
             "schemas", "sg-jira",
         ))
 
-        # Mocks the "current_user_id" method which depends on
-        # introspecting data coming back from the JIRA API and
-        # that we won't simulate for these tests.
-        patcher = mock.patch.object(
```

Comment:
This is the patcher code I was referring to. It's important that the code is unpatched now because `test_integration` will talk to an actual server!